system/addon-manifest.xml

```diff
@@ -29,6 +29,7 @@
   <addon>screensaver.xbmc.builtin.black</addon>
   <addon>screensaver.xbmc.builtin.dim</addon>
   <addon>script.module.pil</addon>
+  <addon>script.module.pycryptodome</addon>
```

Comment:
This should not be there (PIL shouldn't either). It is not a mandatory addon and any addon that actually requires it should import it like normal dependency which should auto enable it.